lib/ethpy/ethpy/base/retry_utils.py

```diff
@@ -55,27 +70,29 @@ async def async_retry_call(
             caller = inspect.stack()[1][3]
             logging.warning(
                 "Retry attempt %s out of %s: Function %s called from %s failed with %s",
-                attempt_number,
+                attempt_number + 1,
                 retry_count,
                 func,
                 caller,
                 repr(exc),
             )
             exception = exc
-            # TODO implement smarter wait here
-            await asyncio.sleep(0.1)
+            await asyncio.sleep(start_latency * backoff_multiplier**attempt_number)
     assert exception is not None
     raise exception
 
 
 def retry_call(
     retry_count: int,
     retry_exception_check: Callable[[Exception], bool] | None,
-    func: Callable[P, R],
-    *args: P.args,
-    **kwargs: P.kwargs,
+    func: Callable[..., R],
+    *args: Any,
```

Comment:
why `Any`, why not `P.args`?

Reply:
Oh I just saw your comment, lemme take a look at that.

Reply:
Oh, I think if we reorg the params it should be ok, try my suggestion above.

Reply:
removed that stuff, is just test now